server/server.js

```diff
@@ -14,11 +14,14 @@ app.listen(port, () => {
 });
 
 app.get('/productDetails', (req, res) => {
-    db.retrieve()
```

Comment:
looks like 4 space indentation

Reply:
Probably don't need to worry about style on removed lines :-P